### PR TITLE
Fix readme installation instructions to use session_exec

### PR DESCRIPTION
**Pull request by pumazi**
... documentation update

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -82,6 +82,21 @@ On a Debian based system, the installation is as follows::
...
     cd ..
 
+Installing the PostgresSQL session_exec extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+    git clone https://github.com/okbob/session_exec
```
Comment (karenc) [inline README.rst]: So it's not necessary to use reedstrm's fork?

Comment (pumazi) [inline README.rst]: My guess is that production is the only one using 9.3 at this point. Maybe I'm hinting at something. ;-)

Comment (karenc) [inline README.rst]: I wasn't able to build session_exec for postgres 9.3 (what we were using on travis) because `session_preload_libraries` is not available for postgres 9.3...  Is it meant to work?

File: README.rst [hunk 2]
```diff
@@ -82,6 +82,24 @@ On a Debian based system, the installation is as follows::
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+
+    git clone https://github.com/okbob/session_exec
+    cd session_exec
+    make USE_PGXS=1 && make USE_PGXS=1 install
```
Comment (karenc) [inline README.rst]: I think `make USE_PGXS=1 install` requires `sudo`.

Comment (pumazi) [inline README.rst]: It may, but that's for the user to determine. This is remaining consistent with the other blocks (e.g. plxslt).

File: README.rst (hunk 1 above)
Comment (pumazi) [inline README.rst]: You're right. This is why reedstrm's fork exists. I added a blob under the code block to note the version difference.

File: README.rst (hunk 2 above)
Comment (karenc) [inline README.rst]: You mean it works without `sudo`, so it's for the user to determine whether to use `sudo` or not?

Comment (pumazi) [inline README.rst]: Depending on how your system is setup and which user you are operating as, Yes.

Comment (reedstrm) [inline README.rst]: A typical linux install will need sudo. A typical OSX developer install (using postgresql.app), perhaps not.

Comment (pumazi) [inline README.rst]: Ok, sudo is probably the way to go here, but it's not within the scope of this pull request. I was just fixing the missing documentation. If you guys want sudo, make another pull request for it. :pineapple:
